### PR TITLE
1.17.1 arguments stuff

### PR DESCRIPTION
**Pull request by apple502j**
Mapped lots of params.

### CODE REVIEW

File: mappings/net/minecraft/command/argument/ScoreboardObjectiveArgumentType.mapping [hunk 1]
```diff
@@ -7,9 +7,17 @@ CLASS net/minecraft/class_2214 net/minecraft/command/argument/ScoreboardObjectiv
 		ARG 1 context
 		ARG 2 builder
 	METHOD method_9391 scoreboardObjective ()Lnet/minecraft/class_2214;
+	METHOD method_9392 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 name
 	METHOD method_9393 getWritableObjective (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/class_266;
 		ARG 0 context
 		ARG 1 name
+	METHOD method_9394 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 name
```
Comment (enbrain) [inline mappings/net/minecraft/command/argument/ScoreboardObjectiveArgumentType.mapping]:
```suggestion
		ARG 0 maxLength
```
This param is the maximum length of an objective name.

File: mappings/net/minecraft/command/argument/CoordinateArgument.mapping [hunk 2]
```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/class_2278 net/minecraft/command/argument/CoordinateArgument
 	FIELD field_10759 MISSING_COORDINATE Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
 	FIELD field_10760 relative Z
 	FIELD field_10761 MISSING_BLOCK_POSITION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
+	FIELD field_32972 TILDE C
```
Comment (liach) [inline mappings/net/minecraft/command/argument/CoordinateArgument.mapping:6]: Maybe document this is a prefix for relative coordinates

File: mappings/net/minecraft/command/argument/ItemStringReader.mapping [hunk 3]
```diff
@@ -9,19 +9,32 @@ CLASS net/minecraft/class_2291 net/minecraft/command/argument/ItemStringReader
 	FIELD field_10807 nbt Lnet/minecraft/class_2487;
 	FIELD field_10808 id Lnet/minecraft/class_2960;
 	FIELD field_10809 cursor I
+	FIELD field_33066 LEFT_CURLY_BRACKET C
+	FIELD field_33067 HASH_SIGN C
```
Comment (liach) [inline mappings/net/minecraft/command/argument/ItemStringReader.mapping:13]: These may respectively be nbt/tag prefix/indicators

File: mappings/net/minecraft/command/argument/LookingPosArgument.mapping [hunk 4]
```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_2268 net/minecraft/command/argument/LookingPosArgument
 	FIELD field_10712 z D
 	FIELD field_10713 y D
 	FIELD field_10714 x D
+	FIELD field_32941 CARET C
```
Comment (liach) [inline mappings/net/minecraft/command/argument/LookingPosArgument.mapping:5]: Same, maybe document this is the prefix for a looking pos

File: mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping [hunk 5]
```diff
@@ -1,12 +1,21 @@
...
+	FIELD field_32185 RIGHT_CURLY_BRACKET C
+	FIELD field_32186 DOUBLE_QUOTE C
 	FIELD field_9898 EXAMPLES Ljava/util/Collection;
 	FIELD field_9899 NOTHING_FOUND_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD field_9900 INVALID_PATH_NODE_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
 	METHOD method_9352 readCompoundChildNode (Lcom/mojang/brigadier/StringReader;Ljava/lang/String;)Lnet/minecraft/class_2203$class_2210;
 		ARG 0 reader
 		ARG 1 name
+	METHOD method_9353 (Lnet/minecraft/class_2487;Lnet/minecraft/class_2520;)Z
+		ARG 1 filter
```
Comment (enbrain) [inline mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping]: This shadows the outer variable.

**Comment by apple502j** [comment]
LOL, happened again :) this is targeted at release branch, @liach

File: mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping [hunk 6]
```diff
@@ -1,12 +1,21 @@
 CLASS net/minecraft/class_2203 net/minecraft/command/argument/NbtPathArgumentType
+	FIELD field_32182 LEFT_SQUARE_BRACKET C
+	FIELD field_32183 RIGHT_SQUARE_BRACKET C
+	FIELD field_32184 LEFT_CURLY_BRACKET C
+	FIELD field_32185 RIGHT_CURLY_BRACKET C
+	FIELD field_32186 DOUBLE_QUOTE C
```
Comment (liach) [inline mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping:6]: These ones seem harder to document indeed

**Review by liach — APPROVED** [review]
Documentation on the role of character constants may be nice, but not needed.

**Review by enbrain — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping (hunk 5 above)
Comment (apple502j) [reply mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping]: Hmm. In this case I believe the outer and inner variables are the same (`computeIfAbsent` iirc?) - how do we map these?

Comment (apple502j) [reply mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping]: Wait, where is this even at? I need to check

Comment (apple502j) [reply mappings/net/minecraft/command/argument/NbtPathArgumentType.mapping]: Oh wait, I looked at the wrong file; nevermind, I'll just un-map this for now

**Comment by liach** [comment]
Ah sorry, but Minecraft releases don't really exist in my perceptions... Which is why I am into yarn and fabric than forge :laughing:

**Review by enbrain — APPROVED** [review]

**Pull request merged**